.github/workflows/argilla.docs.yml

```
@@ -55,16 +57,17 @@ jobs:
         run: echo "${{ github.ref }}"
           echo "${{ github.head_ref }}"
 
+      - run: |
+          pdm run mike deploy latest --push
+          pdm run mike set-default latest
+        if: github.ref == 'refs/heads/main'
+
       - run: pdm run mike deploy dev --push
         if: github.ref == 'refs/heads/feat/v2.0.0'
         # if: github.ref == 'refs/heads/develop'
-        env:
-          GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
 
-      - run: pdm run mike deploy ${{ github.ref_name }} latest --update-aliases --push
+      - run: pdm run mike deploy ${{ github.ref_name }}
```

```suggestion
      - run: pdm run mike deploy ${{ github.ref_name }} latest --push
```
I believe we would still need to push as latstt if we add tags, to ensure we get directed to the latest version.

---

Since the latest version is built from `main` ( a release will be published when new commits are detected in the `main` branch), we can deploy it as the latest one.

Having `latest` linked to tags can introduce errors when creating patch releases for older versions